charts/openyurt/templates/pool-coordinator.yaml

```diff
@@ -0,0 +1,209 @@
+                  - --listen-client-urls=https://0.0.0.0:{{ .Values.poolCoordinator.etcdPort }}
+                  - --cert-file=/etc/kubernetes/pki/etcd/server.crt
+                  - --client-cert-auth=true
+                  - --data-dir=/var/lib/etcd
+                  - --key-file=/etc/kubernetes/pki/etcd/server.key
+                  - --listen-metrics-urls=http://0.0.0.0:12381
+                  - --snapshot-count=10000
+                  - --trusted-ca-file=/etc/kubernetes/pki/etcd/ca.crt
+                image: "{{ .Values.poolCoordinator.etcdImage.registry }}/{{ .Values.poolCoordinator.etcdImage.repository }}:{{ .Values.poolCoordinator.etcdImage.tag }}"
+                imagePullPolicy: {{ .Values.poolCoordinator.etcdImage.pullPolicy }}
+                {{- if .Values.imagePullSecrets }}
+                imagePullSecrets: {{ toYaml .Values.imagePullSecrets | nindent 8 }}
+                {{- end }}
+                name: etcd
+                resources:
+                  {{- toYaml .Values.poolCoordinator.etcdResources | nindent 12 }}
+                startupProbe:
+                  failureThreshold: 24
+                  httpGet:
+                    host: 127.0.0.1
+                    path: /health
+                    port: 12381
+                    scheme: HTTP
+                  initialDelaySeconds: 10
+                  periodSeconds: 10
+                  successThreshold: 1
+                  timeoutSeconds: 15
+                volumeMounts:
+                  - mountPath: /var/lib/etcd
+                    name: etcd-data
+                  - mountPath: /etc/kubernetes/pki/etcd
+                    name: etcd-certs
+            dnsPolicy: ClusterFirst
+            enableServiceLinks: true
+            hostNetwork: true
+            preemptionPolicy: PreemptLowerPriority
+            priority: 2000001000
+            priorityClassName: system-node-critical
+            restartPolicy: Always
+            schedulerName: default-scheduler
+            securityContext:
+              seccompProfile:
+                type: RuntimeDefault
+            terminationGracePeriodSeconds: 30
+            tolerations:
+              - effect: NoExecute
+                operator: Exists
+            volumes:
+              - hostPath:
+                  path: /etc/kubernetes/pki/etcd
+                  type: DirectoryOrCreate
+                name: etcd-certs
+              - emptyDir:
+                  medium: Memory
+                name: etcd-data
+              - hostPath:
+                  path: /etc/ssl/certs
+                  type: DirectoryOrCreate
+                name: ca-certs
+              - hostPath:
+                  path: /etc/pki
+                  type: DirectoryOrCreate
+                name: etc-pki
+              - secret:
+                  secretName: pool-coordinator-certs
+                  defaultMode: 420
+                name: k8s-certs
```

Comment:
Hi, @donychen1134. I am not very familiar with all the certificates of apiserver. Does this means that I need to prepare all the certificates under the `/etc/kubernetes/pki` directory? And other certificates like those related to etcd are managed by the pods themselves.

Reply:
@luc99hen sorry, I forget to add the secret of etcd certs. Etcd container also needs to bind a secret.
I have updated helm chart, please check it again.

Reply:
It seems that the `ca-certs` should be stored in a secret too.

Reply:
Oh, ca-certs and etc-pki are not used in pool-coordinator. They are created by kubeadm. I will delete them.